craft.lic

```diff
@@ -298,8 +298,8 @@ class Craft
     if rank <= 25 # Tier 1  Extremely Easy
       # Buy an abolution sigil and wood totem
       ensure_copper_on_hand(2000, @settings)
-      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 2)
-      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 6)
+      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 2) unless DRCI.exists?("abolition sigil-scroll")
+      DRCT.order_item(crafting_data['artificing'][@hometown]['stock-room'], 6) unless DRCI.exists?("bone totem")
```

Comment:
The comment on line 299 says 'wood totem', but the string here is 'bone totem'. Should these match?

Same with the other totem changes below.

Reply:
I noticed that and not sure why it says wood totem:
```
2021-08-08 11:51:29 -0400:>order
2021-08-08 11:51:29 -0400:You see the following items for sale in the catalog:
2021-08-08 11:51:29 -0400:1).  an intricate induction sigil-scroll...... 625 Kronars
2021-08-08 11:51:29 -0400:2).  an intricate abolition sigil-scroll...... 625 Kronars
2021-08-08 11:51:29 -0400:3).  an intricate congruence sigil-scroll..... 312 Kronars
2021-08-08 11:51:29 -0400:4).  an intricate permutation sigil-scroll.... 625 Kronars
2021-08-08 11:51:29 -0400:5).  an intricate rarefaction sigil-scroll.... 1,500 Kronars
2021-08-08 11:51:29 -0400:6).  a bone totem............................. 125 Kronars
```